utils/big-number/big-number.test.ts

```diff
@@ -0,0 +1,14 @@
+//import { BigNumber } from 'ethers';
+
+import { BigNumber } from 'ethers';
+
+import { fromPositiveNumber } from './index';
+
+describe('Big Number', () => {
+  it('should get the BigNumber with 10', () => {
+    const BG = { _hex: '0x02540be400', _isBigNumber: true } as BigNumber;
+    const positiveNumber = fromPositiveNumber(10, 1);
+
+    expect(positiveNumber._hex).toEqual(BG._hex);
```

Comment:
complex test but yes. 

also this function is curried so u can test that.

fromPositiveNumber(18)(10).toString() === "10000000000000000000" 10 with 18 zeros thats what u gotta test